react/components/Tabs/index.js

```diff
@@ -3,36 +3,54 @@ import PropTypes from 'prop-types'
 
 class Tabs extends PureComponent {
```

Comment:
Deixar como Component. To meio com pé atras com esses nossos PureComponents hehe